UIforETW/UIforETW.rc

```diff
@@ -298,6 +300,17 @@ END
 
 /////////////////////////////////////////////////////////////////////////////
 //
+// AFX_DIALOG_LAYOUT
+//
+
+IDD_UIFORETW_DIALOG AFX_DIALOG_LAYOUT

```

Comment:
It's related to the new dynamic layout support in MFC for VS2015:
https://msdn.microsoft.com/en-us/library/mt270148.aspx
Probably just got added by default since the dialog was modified.